exercises/poker/src/test/java/PokerTest.java

```diff
@@ -0,0 +1,162 @@
+import static org.junit.Assert.assertEquals;
+import java.util.Arrays;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class PokerTest
+{
```

Comment:
This class is looking really good now! :) I would just move this curly bracket to be consistent, so that it's `public class PokerTest {`. If you're using Intellij you could do ctrl + alt + L (on Windows, cmd + alt + L on a mac) which will reformat your code for you. That should fix spacing and formatting issues like this, and you can tell it to optimise your imports as well. [This page](https://www.jetbrains.com/help/idea/2016.3/reformatting-source-code.html) explains it more. I use it all the time because I can never remember to do stuff like remove unused imports manually :)